lib/features/composer/presentation/composer_view.dart

```diff
@@ -413,28 +425,38 @@ class ComposerView extends GetWidget<ComposerController> {
-                          onLoadCompletedEditorAction: controller.onLoadCompletedMobileEditorAction,
-                          onEditorContentHeightChanged: controller.onEditorContentHeightChangedOnIOS,
-                        ),
-                      )),
-                      Obx(() {
-                        if (controller.isContentHeightExceeded.isTrue && PlatformInfo.isIOS) {
-                          return ViewEntireMessageWithMessageClippedWidget(
-                            buttonActionName: AppLocalizations.of(context).viewEntireMessage.toUpperCase(),
-                            onViewEntireMessageAction: controller.viewEntireContent,
-                            topPadding: 12,
-                          );
-                        } else {
-                          return const SizedBox.shrink();
-                        }
-                      }),
-                      SizedBox(height: MediaQuery.viewInsetsOf(context).bottom + 64),
+                      Stack(
+                        children: [
+                          Column(
+                            children: [
+                              Obx(() => Padding(
+                                padding: ComposerStyle.mobileEditorPadding,
+                                child: MobileEditorView(
+                                  arguments: controller.composerArguments.value,
+                                  contentViewState: controller.emailContentsViewState.value,
+                                  onCreatedEditorAction: controller.onCreatedMobileEditorAction,
+                                  onLoadCompletedEditorAction: controller.onLoadCompletedMobileEditorAction,
+                                  onEditorContentHeightChanged: controller.onEditorContentHeightChangedOnIOS,
+                                  onTextSelectionChanged: controller.handleTextSelection,
+                                ),
+                              )),
+                              Obx(() {
+                                if (controller.isContentHeightExceeded.isTrue && PlatformInfo.isIOS) {
+                                  return ViewEntireMessageWithMessageClippedWidget(
+                                    buttonActionName: AppLocalizations.of(context).viewEntireMessage.toUpperCase(),
+                                    onViewEntireMessageAction: controller.viewEntireContent,
+                                    topPadding: 12,
+                                  );
+                                } else {
+                                  return const SizedBox.shrink();
+                                }
+                              }),
+                              SizedBox(height: MediaQuery.viewInsetsOf(context).bottom + 64),
+                            ],
+                          ),
+                          _buildAIScribeSelectionButton(context),
+                        ],
```

Comment:
Should only add the Stack widget and _buildAIScribeSelectionButton when AIScribe is enabled and AI capability is supported.

Reply:
First step done here https://github.com/linagora/tmail-flutter/pull/4197/changes/75758eb93487f840d5cff4fcd1975c0b36cfbf60 and I plan to add the AI capability in a next PR